openslides/users/models.py

```diff
@@ -150,3 +160,83 @@ def reset_password(self, password=None):
         if password is None:
             password = self.default_password
         self.set_password(password)
+
+
+def gen_username(first_name, last_name):
```

Comment:
I thought about bringing model related helper functions together. Like https://github.com/django/django/blob/master/django/contrib/auth/models.py. But I can move these methods to `helpers.py` if you like.